docs/netlify-dev.md

```diff
@@ -136,6 +136,7 @@ Netlify Dev will attempt to detect the site generator or build command that you
 [dev]
   framework = "#custom"
   command = "yarn start" # Command to start your dev server
+  targetPort = 3000 # # The port for your application server, framework or site generator
```

Comment:
When `framework = "#custom"` is configured a use must also add the `targetPort`, otherwise the CLI throws an error.